addons/account/models/account_move.py

```diff
@@ -3334,6 +3334,8 @@ def _amount_residual(self):
                         reconciled = True
                 else:
                     reconciled = True
+            if line.move_id.state in ('draft', 'cancel'):
```

Comment:
Can we not just add this condition to the previous `if` statement?
Why allow it to be set to `true`, then change it back to `false`

Reply:
done